BEIMA.Client/src/pages/Devices/AddDevicePage.js

```diff
@@ -228,6 +229,10 @@ const AddDevicePage = () => {
         let formJSON = '';
         let jsonKey = convertToDbFriendly(formName);
         
+        if(formFields[i].value.length === 0){
+          warnings.push(`${formName} field is empty<br/>`);
```

Comment:
Can we surround {formName} with quotes? And also end with a period.

So instead of:

>Name field is empty

Use:

>"Name" field is empty.

Reply:
Do we want to show every warning or just a generic some fields are empty message instead?

Reply:
Quotes would be good.
As a user I'd prefer to be told exactly what I'm being warned about so I have an informed decision about clicking Proceed or Cancel. if we only mention that some fields are blank it would be user friendly to also highlight them (in yellow instead of red?) like we do with lat/lon so that (again) they know what they're being warned about.

- [ ] add quotes around field names

Reply:
![image](https://user-images.githubusercontent.com/46760776/161394554-a5ec8fb2-01b3-480c-a9a0-63d3c92e21f9.png)